src/store/accounts/nonce.ts

```diff
@@ -20,11 +22,28 @@ export const latestNonceState = atom(get => {
   return get(localNoncesState([network.url, address || '']));
 });
 
+// TODO use AccountsApi.getAccountNonces instead (not yet available in @stacks/blockchain-api-client)
+export const lastApiNonce = atom(async get => {
+  const network = get(currentNetworkState);
+  const address = get(currentAccountStxAddressState);
+  if (!address) return;
+  let addressNonce;
+  try {
+    addressNonce = await fetchAccountNonce(network.url)(address);
```

Comment:
Can't we just `return fetchAccountNonce(network.url)(address);` here?

Reply:
Sure !